src/templates/base/2019/index.html

```diff
@@ -32,7 +32,7 @@
       <h1 class="title title-lg title-alt">{{ self.intro_title() }}</h1>
       <h2>{{ self.intro_sub_title() }}</h2>
       {{ self.mission() }}
-      {% if year < "2020" %}
+      {% if year < "2021" %}
```

Comment:
Is there a way to use the template inheritance and macro overrides to solve this kind of pattern? I'd like to see year-specific templates extend previous years' templates and be the ones to explicitly declare which parts they want to be different.

Reply:
Will have a look. Also did think about removing it completely but thought it might be useful for next year, but yeah you're right - must be a better way to do this.

Reply:
I went a different way with this. - see https://github.com/HTTPArchive/almanac.httparchive.org/pull/1452/commits/7e354c593c11d37e4928b670241f0ea82c5077ef

Means a few more changes but sets us up nicely for 2021